libr/core/cmd.c

```diff
@@ -7086,6 +7332,7 @@ R_API void r_core_cmd_init(RCore *core) {
 		{"*",        "pointer read/write", cmd_pointer, NULL, &pointer_help},
 		{"-",        "open cfg.editor and run script", cmd_stdin, NULL, &stdin_help},
 		{".",        "interpret", cmd_interpret, NULL, &interpret_help},
+		{",",        "create and manipulate tables", cmd_table, NULL, &table_help},
```

Comment:
this probably cannot work as `table_help` is not of type `RCmdDescHelp`.

Reply:
oh right, my bad.fixed